image.yaml

```diff
@@ -44,11 +44,10 @@ modules:
       repositories:
```

Comment:
I think we should start using more intelligent repository/module names, especially since we're starting from scratch.  Maybe, jboss.eap72.standalone.repo for the repository name and jboss.eap72.standalone.product-installer for the module? If you've got better name, feel free to use them.

Reply:
Do you think we'd gain from using the version in the module / repo name? Since this image will generally move forward with whatever version of EAP we're delivering, it may be 7.2 for a while, but then moving to 7.3 we'd keep the same modules and repos and just bump the artifact.

Maybe this is too optimistic from my POV,

jboss.eap.cd.standalone.repo | jboss.eap.cd.standalone.product-installer maybe.

Standalone is another overloaded one when it comes to EAP, since we usually divide the appserver itself into standalone vs domain. I'll think about this a little. I do agree the names can be improved.

Reply:
Yeah, i don't know what benefit there would be, especially since 72 is in the branch name.

As for standalone, maybe base instead?  I've never really liked the term either.  It's really just the raw, unadulterated product, as opposed to the -openshift image, which has all the tweaks for running in OpenShift.  Maybe jboss.eap.product.cp?  Does cd need to be in there?  Maybe jboss.eap.product.image.repo and jboss.eap.openshift.image.repo, yielding things like: jboss.eap.product.image.installer and jboss.eap.openshift.image.launch?  Maybe image is redundant, so just jboss.eap.product.installer, jboss.eap.openshift.launch?  (I hate naming things.)